cmd/helm/list_test.go

```diff
@@ -56,9 +56,7 @@ func TestListCmd(t *testing.T) {
 			rels: tt.resp,
 		}
 		cmd := newListCmd(c, &buf)
-		for flag, value := range tt.flags {
-			cmd.Flags().Set(flag, value)
-		}
+		setFlags(cmd, tt.flags)
```

Comment:
flags can be added as args, I deleted the flags arg a while ago, it is a zombie that cannot be killed